Automattic/themes
# Blockbase: Update the package.json

#### Changes proposed in this Pull Request:
This tries to align the package.json for Blockbase with the same files in Quadrat and Skatepark, so that we have a more unified approach.

This will be useful as we move towards generated themes.

To test:

Remove node_modules in blockbase
Run npm i
Run npm run build
Check that the source map still works as before.
